ProjectSettings/SceneTemplateSettings.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
What is this file for?

Reply:
No clue, unity added it when I added the images for Adam, so I'll take it.

Reply:
Considering it is animation related, no, it can stay.

Reply:
It is not animation related, it is a scene template for dependencies and I don't see the need for this in the project.

Reply:
If unity makes it, it gets added if it isn't gitignored. IMO it's fine to stay.

Reply:
Unity made this because you opened those settings in the project settings but this is not necessary at all and can be remove, I got that file by testing around and in all older versions its not there so not needed.

Reply:
I still see no reason to remove it.

Reply:
There is no reason to keep this either and it is never used so to keep the project clean please remove this file.

Reply:
Could this please be resolved, so I can start recording for the game trailer?

Reply:
If a file is known to be useless it should be deleted. Even if it already made a previous PR. Keep clutter down as much as possible please. No need to retroactively clean the entire code base but if you **do** run into something just clean it...

Reply:
I stand by the position that autogenerated unity files are **always** accepted in, no questions asked. I do not intend to deviate from that here. The only exception I can see to that point of view is when adding the file **actively** breaks it, which is not the case here.

Reply:
Not wanting to spend time validating the usefullness of each file is totally fine, it is not a priority. But if a file is known useless why take such a hard stance that it must stay? Just delete the junk and the conflict is solved. Then these features can be used for Expo 1.

Reply:
I’ve explained my standpoint multiple times, I’m not going to do that again. Take this, or don’t, really don’t care. If we went the original planning as demanded by the group we wouldn’t have this feature at all anyway so that argument can go straight into the trash can.

Reply:
The reason we wouldn't have that feature in the first place was because it wasn't completed in sprint 5 as there was no clearly defined feature requirement list for the Expo. But it was you that said you really wanted these features included in the Expo demo to begin with. 

Besides, getting these features merged intime is not the argument,  It would be a nice benefit of resolving this issue quickly, but as that apparently is not going to happen we can just focus on the main problem. 
On which I agree with @Westlanderz  that it is bad practice to include known junk based on some undefined policy of “All auto generated files are included!”. So unless you have a good reason for this policy my vote goes to simply deleting the file...

Reply:
Why yes I do have a good reason. It does no harm at all, and imo it doesn’t clutter the repo so it is fine. I take way more issue with unity generating a shitload of meta files but unfortunately that can’t be changed.

Reply:
But the `meta` files are necessary to run the project this file is not. And when multiple other individuals say it clutters the repo in their opinion, we all should just follow that if the file does not impact anything.

Reply:
Did you read what I wrote at all? I never said I wanted them gone, I know they are needed but if there is any clutter in this repo it would be that. Furthermore, as we don’t seem to get anywhere with this PR, and considering it’s past the feature freeze date, I suggest we leave this as is for now, and revisit this for the next session. I still stand for its inclusion, and I disagree with any kind of clutter reason as it doesn’t clutter, it is nicely stored away in its own folder. And @Duma0, it never was an undefined policy, on multiple occasions new unity asset files have been approved without any second thought at all, no complains whatsoever, and I see no reason to change that policy now, especially since the file does nothing wrong. I like having all the project files available, and I mean _all_ of them, as it gives maximum reproducibility.

Reply:
After a group vote, 6-2 were in favor of keeping the file. I've reflected this decision and the PR is ready for another review.